src/styles/theme.scss

```diff
@@ -1,100 +1,204 @@
-  --color-interaction-secondary-alt-disabled:#E7E7E7;
-  --color-semantic-info:#000000;
-  --color-semantic-info-hover:#000000;
-  --color-semantic-info-active:#000000;
-  --color-semantic-disabled:#000000;
-  --color-semantic-success:#1fd528;
-  --color-semantic-success-hover:#000000;
-  --color-semantic-success-active:#000000;
-  --color-semantic-success-disabled:#000000;
-  --color-semantic-warning:#000000;
-  --color-semantic-warning-hover:#000000;
-  --color-semantic-warning-active:#000000;
-  --color-semantic-warning-disabled:#000000;
-  --color-semantic-critical:#cd1919;
-  --color-semantic-critical-hover:#000000;
-  --color-semantic-critical-active:#000000;
-  --color-semantic-critical-disabled:#000000;
-  --color-text-main:#222222;
-  --color-text-subtle:#8F8F8F;
-  --color-text-reverse:#FFFFFF;
-  --color-text-reverse-subtle:#E7E7E7;
-  --color-text-disabled:#AAAAAA;
-  --color-text-info:#000000;
-  --color-text-success:#000000;
-  --color-text-warning:#000000;
-  --color-text-critical:#000000;
-  --color-border-1:#E6E6E6;
-  --color-border-2:#F9F9F9;
-  --color-neutral-0:#FFFFFF;
-  --color-neutral-1:#F5F5F5;
-  --color-neutral-2:#EDEDED;
-  --color-neutral-3:#E7E7E7;
+  --color-interaction-primary-hover: #170696;
+  --color-interaction-primary-active: #170696;
+  --color-interaction-primary-disabled: rgba(176, 176, 176, 0.5);
+  --color-interaction-primary-alt: #FFFFFF;
+  --color-interaction-primary-alt-hover: #E8EFFD;
+  --color-interaction-primary-alt-active: #CBD8F0;
+  --color-interaction-primary-alt-disabled: #E7E7E7;
+
+  /**
+  SECONDARY INTERACTION
+  **/
+  --color-interaction-secondary: #ffffff;
+  --color-interaction-secondary-hover: #d3d3d3;
+  --color-interaction-secondary-active: #CCCCD1;
+  --color-interaction-secondary-disabled: rgba(176, 176, 176, 0.5);
+  --color-interaction-secondary-alt: #FFFFFF;
+  --color-interaction-secondary-alt-hover: #E8E8E8;
+  --color-interaction-secondary-alt-active: #D9D9D9;
+  --color-interaction-secondary-alt-disabled: #E7E7E7;
+
+  /**
+  SEMANTIC
+  **/
+  --color-semantic-info: #0073bd;
+  --color-semantic-info-hover: #1c84cc;
+  --color-semantic-info-active: #3895d8;
+  --color-semantic-disabled: #6f6f6f;
+  --color-semantic-success: #0c884c;
+  --color-semantic-success-hover: #229a60;
+  --color-semantic-success-active: #39ab74;
+  --color-semantic-success-disabled: #767676;
+  --color-semantic-warning: #a86500;
+  --color-semantic-warning-hover: #b8771c;
+  --color-semantic-warning-active: #c78938;
+  --color-semantic-warning-disabled: #747474;
+  --color-semantic-critical: #d32947;
+  --color-semantic-critical-hover: #de3a57;
+  --color-semantic-critical-active: #e84d68;
+  --color-semantic-critical-disabled: #6e6e6e;
+  --color-semantic-critical-bg-alt: #f9e1e5;
+	--color-semantic-neutral: #222222;
+	--color-semantic-neutral-hover: #3a3a3a;
+	--color-semantic-neutral-active: #535353;
+	--color-semantic-neutral-inactive: #6e6e6e;
```

Comment:
nit: Indentation. We should probably get basic linting and prettier checks setup in a github workflow for this repo.